mappings/net/minecraft/loot/function/AndFunction.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5641 net/minecraft/loot/function/AndFunction
```

Comment:
Should this be renamed `AndLootFunction` to match others?

Reply:
Ah yeah.